### DISCUSSION TITLE
is the website down?

### DISCUSSION THREAD

**Discussion by dlebauer** [kerhub/issue-forms-creator#30] [Q&A]
This looks interesting but the links to https://www.issue-forms-creator.app/new is broken. Is this a typo, or is the site down?

**Comment by geromegrignon** [comment]
Hi @dlebauer, I have to deploy it to GitHub pages.
I decided not to renew domain bills for issue-forms-creator.app :)

**Comment by geromegrignon** [comment]
Deployed here: https://issue-forms-creator.netlify.app/
